web/src/AddNeighbourhoodMode.svelte

```diff
@@ -225,8 +225,16 @@
     drawnBoundary = null;
     neighbourhoodBoundary = null;
   }
+
+  function onKeyDown(e: KeyboardEvent) {
+    if (e.key == "Enter" && neighbourhoodBoundary) {
```

Comment:
I always instinctively try to press enter here to confirm and get the naming prompt